### PR TITLE
New version and change in maintainer

### PR DESCRIPTION
**Pull request by emiliom**
Checklist
* [x ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -64,12 +64,13 @@ test:
...
+  license_file: LICENSE
   summary: 'Clean, simple, and fast access to public hydrology and climatology data'
   doc_url: http://ulmo.readthedocs.io/
   dev_url: https://github.com/ulmo-dev/ulmo/
 
 extra:
   recipe-maintainers:
+    - emiliom
     - ocefpaf
-    - dharhas
```
Comment (ocefpaf) [inline recipe/meta.yaml:75]: @dharhas do we have your :+1: here?

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (emiliom) [reply recipe/meta.yaml:75]: FYI, I became the maintainer of ulmo ~ 18 months ago

Comment (ocefpaf) [reply recipe/meta.yaml:75]: This is about the conda-forge feedstock. We cannot remove someone without their OK.

PS: congrats on the commit rights upstream!

**Comment by dharhas** [comment]
Good with me. I handed over maintenance a while back.

**Comment by emiliom** [comment]
Thanks @ocefpaf!

**Pull request merged**